elrond-config.md

```diff
@@ -414,18 +416,36 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
 ```k
     syntax InternalCmd ::= "#endWasm"
  // ---------------------------------
-    rule <commands> #endWasm => popCallState ~> dropWorldState ... </commands>
+    rule <commands> #endWasm => #asyncExecute ~> #setVMOutput ~> popCallState ~> dropWorldState ... </commands>
          <instrs> .K </instrs>
-         <out> OUT </out>
-         <logs> LOGS </logs>
-         <vmOutput> _ => VMOutput( OK , .Bytes , OUT , LOGS) </vmOutput>
       [priority(60)]
 
     syntax InternalCmd ::= "#waitWasm"  [klabel(#waitWasm), symbol]
  // ----------------------------------
     rule <commands> #waitWasm => .K ... </commands>
          <instrs> .K </instrs>
       [priority(60)]
+
+    syntax InternalInstr ::= "#waitCommands"
```

Comment:
Would it be hard to add somewhere a description of what these wait commands do and now they interact with each other? Or a description of how execution works, given that we have three execution cells (<k>, <commands> and <instrs>)?